raft/util.go

```diff
@@ -93,3 +93,18 @@ func DescribeEntry(e pb.Entry, f EntryFormatter) string {
 	}
 	return fmt.Sprintf("%d/%d %s %s", e.Term, e.Index, e.Type, formatted)
 }
+
+func limitSize(ents []pb.Entry, maxSize uint64) []pb.Entry {
```

Comment:
If the last entry is 10MB, it may exceed at least 6 MB. I think the user deserves a warning or log for it.